content/SCALE/GettingStarted/FirstTimeLogin.md

```diff
@@ -104,25 +104,25 @@ The Task Manager displays all running and failed jobs/processes.
 
 ![TaskManagerSCALE](/images/SCALE/TaskManagerSCALE.png "TrueNAS SCALE Task Manager")
 
-Users can click the *History* button to open the **Jobs** screen. **Jobs** lists all *Successful*, *Active*, and *Failed* jobs. Users can also click *View Logs* next to a failed process to view its log information and error message.
+Users can click the **History** button to open the **Jobs** screen. **Jobs** lists all Successful, Active, and Failed jobs. Users can also click **View Logs** next to a failed process to view its log information and error message.
 
 ![TaskManagerHistorySCALE](/images/SCALE/TaskManagerHistorySCALE.png "Task Manager History")
 {{< /tab >}}
 
 {{< tab "Alerts" >}}
-The *Alerts* button displays the *Alerts* menu, which shows all current alerts. Users may dismiss alerts individually, or all at once.
+The **Alerts** button displays the **Alerts** menu, which shows all current alerts. Users may dismiss them individually or all at once.
 
-The *Alerts* menu also lets users configure *Alert Settings*, *Alert Services*, and *Email*.
+The **Alerts** menu also lets users configure **Alert Settings**, **Alert Services**, and **Email**.
 
 ### Alert Settings
 
-The *Alert Settings* screen has options for setting the warning level and frequency for alerts specific to application actions.
+The **Alert Settings** screen has options for setting the warning level and frequency for alerts specific to application actions.
 
 ![AlertSettingsSCALE](/images/SCALE/AlertSettingsSCALE.png "TrueNAS SCALE Alert Settings")
 
-The *Set Warning Level* drop-downs customize alert importance. Each warning level has an icon and color to express its urgency.
+The **Set Warning Level** drop-downs customize alert importance. Each warning level has an icon and color to express its urgency.
```

Comment:
possessive case and the debate about dropdown verses drop-down (tbd). On "the level of urgency" it could also be "urgency level"
```suggestion
The **Set Warning Level** drop-downs customize alert importance. Each warning level has an icon and color to express the level of urgency.
```